business_requirement_deliverable/report/br_deliverable_report.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import tools
+from openerp import fields, models
+
+
+class DeliverableReport(models.Model):
+    _name = "deliverable.report"
+    _description = "Deliverable Report"
+    _auto = False
+
+    name = fields.Char('Name', readonly=True)
+    description = fields.Char('Description', readonly=True)
+    partner_id = fields.Many2one('res.partner', 'Customer',
+                                 readonly=True)
+    project_id = fields.Many2one('project.project', 'Master Project',
+                                 readonly=True)
+    change_request = fields.Boolean('Change Request?', readonly=True)
+    priority = fields.Selection([('0', 'Low'), ('1', 'Normal'),
+                                 ('2', 'High')],
+                                'Priority', readonly=True)
+    dlv_product = fields.Many2one('product.product', 'Dlv Product',
```

Comment:
better this order:
    dlv_description = fields.Text('Deliverable Description', readonly=True)
    dlv_product = fields.Many2one('product.product', 'Dlv Product',
                                  readonly=True)
    res_description = fields.Text('Resource Description', readonly=True)
    res_product = fields.Many2one('product.product', 'Res Product',
                                  readonly=True)